RAT_README.txt

```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------------
+This readme contains the list of excluded files that don't have apache headers and explanations on why they don't and cannnot have headers 
+To run the RAT report for Trafodion:
+1. Install Apache Rat. Taht will install the needed jar file. 
+2. Use command line:
+   java -jar apache-rat-0.11.jar  -E $MY_SQROOT/../../.rat-excludes -d $MY_SQROOT/../.. > <RAT output log file>
+------------------------------------------------------------------------------
+FILE           :                      NOTES
+------------------------------------------------------------------------------
+          
+          
+.gitignore  : File contains pattterns that GIT reads. Cannot have comments 
+.rat-excludes   : This file used by RAT cannot have a comment section 
+package.exclude* : List of files to exclude from packaging - cannot pargse comments.      
+eclipse.*project* : Eclipse related project file
+*.csv     :  Macro file cannot have comments.
+*/hbase-trx/hbase-protocol/* : Copied from Hbse and cannot haveApache copyrights
```

Comment:
Corrected typo.  Rewrite the comment  that was confusing about the protobuf files. Removed 2 unneeded files that had previously been copied and weren't needed.